### PR TITLE
Dark mode tweaks for timeline.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
- Modify UI/GPU color palettes to match Terry's changes in #552 
- Use contrastForegroundWhite instead of jarring white #FFFFFF
- General cleanup
Screen Shot 2019-04-19 at 8 26 52 AM

### CODE REVIEW

File: packages/devtools/lib/src/timeline/event_details.dart [hunk 1]
```diff
@@ -39,7 +40,10 @@ class EventDetails extends CoreElement {
   }
 
   static const defaultTitleText = '[No event selected]';
-  static const defaultTitleBackground = Color(0xFFF6F6F6);
+  static const defaultTitleBackground = ThemedColor(
+    Color(0xFFF6F6F6),
+    Color(0xFF323232),
```
Comment (terrylucas) [inline packages/devtools/lib/src/timeline/event_details.dart]: Does this color map to a real color in the Material Dark color or some offset of a base?

**Comment by terrylucas** [comment]
The timeline dark mode is quite appealing.

File: packages/devtools/lib/src/timeline/event_details.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools/lib/src/timeline/event_details.dart]: This color is the same as the "lighter" background section in the flame chart (the background of the UI events. I'll change both to use Grey900 +2 from the bottom of Material Dark Theme page. Maybe we should change the background of the plotly chart to match?

Comment (terrylucas) [reply packages/devtools/lib/src/timeline/event_details.dart]: Seems like the chart and other background sections of the flames charts should match.  Shouldn't all our pages (memory chart too) all be the same?

**Comment by kenzieschmoll** [comment]
Updated color to use Material Dark Grey900+2. Terry, I changed the chart background color (Memory and FPS) to match. Updated the PR screenshot.

**Comment by terrylucas** [comment]
Looks nice

**Pull request merged**